host/tests/dsDisplay_L3_Tests/dsDisplay_test03_GetDisplayAspectRatio.py

```diff
@@ -0,0 +1,181 @@
+
+    def testRunPrerequisites(self):
+        """
+        Runs Prerequisite commands listed in test-setup configuration file on the dut.
+
+        Args:
+            None.
+        """
+
+        #Run test specific commands
+        test = self.testSetup.get("assets").get("device").get(self.testName)
+        cmds = test.get("execute")
+        if cmds is not None:
+            for cmd in cmds:
+                self.writeCommands(cmd)
+
+    #TODO: Current version supports only manual verification.
+    def testGetDisplayAspectRatio(self, manual=False, port="dsVIDEOPORT_TYPE_HDMI" , index:int = 0):
+        """
+        Gets the aspectratio of the display.
+        Args:
+            manual (bool, optional): Manual verification (True: manual, False: other verification methods).
+                                    Defaults to other verification methods
+                                    port (str, optional): The type of video port to retrieve the EDID Bytes.
+                                    index (int, optional): The index of the port to retrieve the EDID Bytes.
+        Returns:
+            bool
+        """
+        # Get the aspectratio from the display
+        aspectratio = self.testdsDisplay.getAspectRatio(port, index)
+
+        if manual:
+            # Manual verification
+            return self.testUserResponse.getUserYN(f"Is the aspectratio {aspectratio} correct? (Y/N):")
+        else:
+            # Automation verification
+            #TODO: Add automation verification methods
+            return False
+
+    def testFunction(self):
+        """
+        This function will test the Display by getting the aspectratio of the display.
+
+        This function:
+        - Downloads the required assets.
+        - Runs the prerequisite commands.
+        - Retrieves aspectratio for each supported port and verifies them.
+        - Cleans up assets after the test.
+
+        Returns:
+            bool: Final result of the test.
+        """
+
+        # Download the assets listed in the test setup configuration file
+        self.testDownloadAssets()
+
+        # Run Prerequisites listed in the test setup configuration file
+        self.testRunPrerequisites()
+
+        # Create an instance of the dsDisplayClass
+        self.testdsDisplay = dsDisplayClass(self.deviceProfile, self.hal_session)
+
+        # Initialize the dsDisplay module
+        self.testdsDisplay.initialise()
+
+        self.log.testStart(self.testName, '1')
+
+        # Get the supported video ports
+        supported_ports = self.testdsDisplay.getSupportedPorts()
+        result=True
+
+        if not supported_ports:
+            self.log.error("No supported ports found.")
+            return False
+
+        # Loop through the supported video ports
+        for port, index in supported_ports:
+            self.log.info(f"Testing port {port} at index {index}.")
+            edid_result = self.testGetDisplayAspectRatio(manual=True, port=port, index=index)
```

Comment:
prefix the aspect ratios to be tested as 16:9 and 4:3. Both have to be validated with connected display. Ask and wait for the user to switch the aspect ratio and then verify the value with get function